setup/.abcde.conf

```diff
@@ -91,8 +92,8 @@ mungefilename ()
 }
 
 # What extra options?
-MAXPROCS=2                              # Run a few encoders simultaneously
+MAXPROCS=4                              # Run a few encoders simultaneously
 PADTRACKS=y                             # Makes tracks 01 02 not 1 2
 EXTRAVERBOSE=2                          # Useful for debugging
 COMMENT='abcde version 2.8.1'           # Place a comment...
-EJECTCD=y                               # Please eject cd when finished :-)
+EJECTCD=y                               # Please eject cd when finished :-)
```

Comment:
I'm excited to test this out. I had quite a lot of problems with disk recognition. And I was thinking that we might want to update the documentation to include finding the offset of your specific drive.

Reply:
One of the main problems for ARM with Music CD's currently, the master branch is set to use CDDB but they have long since closed their doors. They closed around the same time Microsoft shut down their CRC metadata service.

This updates the config to use MusicBrainz and also adds the Musicbrains module to ARM allowing the Disc to be correctly identified in both ARM and ARMui.
Musicbrains do have a searchable database, via API that could allow users to search for disc, but because musicbrainz use a specific way of hashing each disc i didnt find it ever necessary to manually search.